core/src/main/scala/fs2/Scheduler.scala

```diff
@@ -0,0 +1,47 @@
+package fs2
+
+import scala.concurrent.duration.FiniteDuration
+import java.util.concurrent.{ Executors, ScheduledExecutorService, TimeUnit }
+
+/** Provides the ability to schedule evaluation of thunks in the future. */
+trait Scheduler {
+
```

Comment:
@pchlupacek Completely agree. I just pushed a fix for this -- let me know what you think.